src/neo/Network/P2P/Payloads/OracleResponse.cs

```diff
@@ -0,0 +1,73 @@
+using Neo.IO;
+using Neo.IO.Json;
+using Neo.Ledger;
+using Neo.Persistence;
+using Neo.SmartContract.Native;
+using Neo.SmartContract.Native.Oracle;
+using Neo.VM;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class OracleResponse : TransactionAttribute
+    {
+        private const int MaxResultSize = 1024;
```

Comment:
https://github.com/neo-project/neo-vm/blob/87d767552d4f0cdc9a2c88cdec560d177dc253c6/src/neo-vm/ExecutionEngine.cs#L32

```suggestion
        private const int MaxResultSize = 1024 * 1024;
```

Reply:
The result is filtered. So `1024` is enough.